.drone.jsonnet

```diff
@@ -123,6 +123,7 @@ local PipelinePostgreSQL(test_set) = Pipeline(
 	PipelineSQLite("conversation-2"),
 	PipelineSQLite("federation"),
 	PipelineSQLite("reaction"),
+	PipelineSQLite("message_expire"),
```

Comment:
As mentioned before, please move your file to chat/ so we don't need a new pipeline (saves ~2mins of overhead